src/main/java/kohn/rx/votesmart/VoteSmartController.java

```diff
@@ -0,0 +1,97 @@
+package kohn.rx.votesmart;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import io.reactivex.Observable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+
+public class VoteSmartController {
+
+    private Provider<VoteSmartView> viewProvider;
+    private VoteSmartService service;
+    private Disposable disposable;
+
+    @Inject
+    public VoteSmartController(VoteSmartService service, Provider<VoteSmartView> viewProvider) {
+        this.viewProvider = viewProvider;
+        this.service = service;
+    }
+
+    public void loadStates() {
+        disposable = Observable.interval(0, 30, TimeUnit.SECONDS)
```

Comment:
Does this need to refresh every 30 seconds? Are you waiting for new States to added to the country?

Copying code without thinking is really problematic.

Reply:
I believe I initially had it to refresh every 10 days, tried with filters and without, debugging didn't get me anywhere either, and this was just the last thing I tried.